Gemfile

```diff
@@ -30,6 +30,7 @@ gem 'jbuilder'
 gem 'sidekiq'
```

Comment:
Te pongo el comentario en la 1º linea para que lo veas xD
- Solo tiene la lógica de generación y consulta de pagos
- Dentro de los pagos, podríamos considerar (lo dejé así en el código) cuando "se captura" el pago por mas que esté in process así el chango lo puede usar.
- La parte de bloqueo y generación del link de pago, me parece "mejor" hacerlo en un review aparte así no se dilata tanto este. (por mas que no usemos este código...)
- Agregué una columna nueva para saber hasta cuando tiene pagada la licencia (y así checkear si hay un nuevo pago) ya que el subscribed_at, es como para saber cuando se termina la suscripción y no hasta cuando está pagada.
- FALTA el task que checkea los IPNs, consulta pagos y activa licencias. No se si te parece bien hacerlo acá o después (ya que de momento no va a hacer nada)

Reply:
Funcionó, lo leí =P Voy con los puntos en orden:

- Ok, me parece bien.
- También me parece bien. Igualmente, lo marcaría de alguna manera como "a revisar" así le damos una pasada al menos un día después y ahí si todavía no pasa nada nos vemos.
- Ok, estoy de acuerdo.
- Perfecto.
- Mejor en otro PR, este ya está al límite de la gordura =P
- Bien también, cuando lleguemos a esa parte que así sea.